plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -761,6 +776,14 @@ public void setDeviceType(DeviceType deviceType) {
             _deviceType = deviceType;
         }
 
+        public boolean getIothreads() {
```

Comment:
This returns a boolean, but to me it implies a number which it will return.

Is this method even ever called?